Doc/library/pdb.rst

```diff
@@ -639,6 +639,52 @@ can be overridden by the local file.
 
    Print the return value for the last return of the current function.
 
+.. pdbcommand:: exceptions [arg]
+
+   List or jump between chained exceptions.
+
+   If ``Pdb.post_mortem()`` is called with a chained exception, it will now allow
```

Comment:
Maybe mention `pm()` as well? It's probably a more common entry for this command.